### PR TITLE
feat: BNGP-4483 Limit number of characters in input fields

### PR DESCRIPTION
**Pull request by KiranVarma02**

### CODE REVIEW

File: packages/webapp/src/routes/land/add-landowner-individual.js [hunk 1]
```diff
@@ -38,12 +39,14 @@ const handlers = {
 
     const firstNameError = validateFirstLastName(firstName, 'first name', 'firstNameId')
     const lastNameError = validateFirstLastName(lastName, 'last name', 'lastNameId')
+    const middleNameError = validateLengthOfCharsLessThan50(middleName, 'middle name', 'middleNameId')
 
     if (!isEmpty(firstNameError) || !isEmpty(lastNameError)) {
```
Comment (paulsimonandrews) [inline packages/webapp/src/routes/land/add-landowner-individual.js:44]: Need to add the check for `!isEmpty(middleNameError)` here too

**Review by paulsimonandrews — COMMENTED** [review]
Noticed an issue with middle name error just in case you've not seen it

File: packages/webapp/src/routes/land/add-landowner-individual.js (hunk 1 above)
Comment (KiranVarma02) [reply packages/webapp/src/routes/land/add-landowner-individual.js:44]: middle name can be empty, we only want this check for first and last name.

Comment (KiranVarma02) [reply packages/webapp/src/routes/land/add-landowner-individual.js:44]: I get it now :), I will add it

Comment (KiranVarma02) [reply packages/webapp/src/routes/land/add-landowner-individual.js:44]: done

**Review by pwadmore-ea — CHANGES_REQUESTED** [review]
package/webapp/src/routes/add-landowner-organisation-conservation-covenan accepts organisation names greater than 50 characters. The data in the screenshot below can be saved.

image

I have included more detail in Jira.

File: packages/database-version-control/package.json [hunk 2]
```diff
@@ -10,7 +10,7 @@
     "local:install": "bin/init-database-version-control-env-if-needed.sh",
     "local:cli-up": ". ../../docker/secrets/DATABASE_VERSION_CONTROL_ENV && node src/migrator-cli.js up",
     "cli-up": "node src/migrator-cli.js up",
-    "local:cli-down": ". ../../docker/secrets/DATABASE_VERSION_CONTROL_ENV && node src/migrator-cli.js down",
+    "local:cli-dowlocal:cli-upn": ". ../../docker/secrets/DATABASE_VERSION_CONTROL_ENV && node src/migrator-cli.js down",
```
[the remark this answers is not in the document]
Comment (KiranVarma02) [reply packages/database-version-control/package.json:13]: done

File: packages/webapp/src/routes/land/add-landowner-individual.js [hunk 3]
```diff
@@ -38,12 +39,14 @@ const handlers = {
 
     const firstNameError = validateFirstLastName(firstName, 'first name', 'firstNameId')
     const lastNameError = validateFirstLastName(lastName, 'last name', 'lastNameId')
+    const middleNameError = validateLengthOfCharsLessThan50(middleName, 'middle name', 'middleNameId')
 
-    if (!isEmpty(firstNameError) || !isEmpty(lastNameError)) {
+    if (!isEmpty(firstNameError) || !isEmpty(lastNameError) || !isEmpty(middleNameError)) {
       return h.view(constants.views.ADD_LANDOWNER_INDIVIDUAL, {
         err: Object.values({ ...firstNameError, ...lastNameError }),
```
[the remark this answers is not in the document]
Comment (KiranVarma02) [reply packages/webapp/src/routes/land/add-landowner-individual.js:46]: done

**Review by paulsimonandrews — DISMISSED** [review]
LGTM

**Pull request merged**